flatland_plugins/include/flatland_plugins/imu.h

```diff
@@ -0,0 +1,102 @@
+/*
+ *  ______                   __  __              __
+ * /\  _  \           __    /\ \/\ \            /\ \__
+ * \ \ \L\ \  __  __ /\_\   \_\ \ \ \____    ___\ \ ,_\   ____
+ *  \ \  __ \/\ \/\ \\/\ \  /'_` \ \ '__`\  / __`\ \ \/  /',__\
+ *   \ \ \/\ \ \ \_/ |\ \ \/\ \L\ \ \ \L\ \/\ \L\ \ \ \_/\__, `\
+ *    \ \_\ \_\ \___/  \ \_\ \___,_\ \_,__/\ \____/\ \__\/\____/
+ *     \/_/\/_/\/__/    \/_/\/__,_ /\/___/  \/___/  \/__/\/___/
+ * @copyright Copyright 2017 Avidbots Corp.
+ * @name	diff_drive.h
```

Comment:
Should be `imu.h` and "IMU plugin" in the brief.
Also, should we be using a newer set of dates and authors here?

Reply:
I'll update this to `imu.h` and update the brief. But yeah as for the other stuff, I'll do whatever you'd like. My role here was taking an old PR and bringing it up to date with a bunch of merge conflict resolution. But I'm not the original author.

Reply:
Okay we can leave the date and author as-is then.